artist-core/src/main/kotlin/com/uber/artist/KotlinArtistCodeGenerator.kt

```diff
@@ -53,7 +53,7 @@ class KotlinArtistCodeGenerator : ArtistCodeGenerator<FileSpec, TypeSpec.Builder
   override fun generateTypeSpecFor(stencil: KotlinViewStencil, rPackageName: String, traitMap: Map<Class<out KotlinTrait>, KotlinTrait>, superinterfaceClassName: String?): TypeSpec.Builder {
     val rClass = ClassName(rPackageName, "R")
     val typeBuilder = TypeSpec.classBuilder(stencil.name())
-        .addModifiers(KModifier.PUBLIC, KModifier.OPEN)
```

Comment:
what is redundant? does kotlinpoet assume open methods should be public?

Reply:
public is the default in kotlin